docs/pages/setup/guides/ec2-tags.mdx

```diff
@@ -1,20 +1,26 @@
 ---
-title: EC2 Tags as Teleport node labels
-description: How to setup Teleport node labels based on EC2 tags
-h1: Sync EC2 Tags and Teleport node labels
+title: EC2 Tags as Teleport Node Labels
```

Comment:
Just a note - we'll be adding a first-class feature to Teleport that does this in Teleport 10 or 11, at which point this guide will become deprecated.

Reply:
Created an issue to remove this when needed: https://github.com/gravitational/teleport/issues/11929